terraform/modules/enclave/prometheus/test/fixtures/main.tf

```diff
@@ -0,0 +1,75 @@
+provider "aws" {
+  region = "eu-west-2" # "${var.aws_region}"
+}
+
+terraform {
+  required_version = ">= 0.11.7"
```

Comment:
We might want to pin this exactly to 0.11.7 like we do for all the rest of the codebase